.vscode/tasks.json

```diff
@@ -0,0 +1,25 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "0.1.0",
+    "command": "grunt",
```

Comment:
I added this file mostly for this option -- it lets you easily run our grunt tasks from within VSCode.

Maybe we should add the `.vscode` directory to `.gitignore` so that everyone can tweak this individually?